dashboard/src/components/OperatorView/OperatorCapabilityLevel.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { shallow } from "enzyme";
+import * as React from "react";
+import CapabiliyLevel from "./OperatorCapabilityLevel";
+
+[
+  {
+    name: "basic install level",
+    expectedLevels: 1,
+    level: "Basic Install",
```

Comment:
I was surprised that these capability level's are not identifiers (but checked upstream and they're not). But perhaps they should at least be constants in the module?